webapi/src/main/scala/org/knora/webapi/responders/v2/ResourcesResponderV2.scala

```diff
@@ -959,23 +989,31 @@ class ResourcesResponderV2(responderData: ResponderData) extends ResponderWithSt
       * @param resourceIris   the resources to query for.
       * @param versionDate    if defined, requests the state of the resources at the specified time in the past.
       * @param requestingUser the the client making the request.
+      * @param schemaOptions  the schema options submitted with the request.
       * @return a [[ReadResourcesSequenceV2]].
       */
     private def getResourcesV2(resourceIris: Seq[IRI],
                                propertyIri: Option[SmartIri] = None,
                                versionDate: Option[Instant] = None,
+                               targetSchema: ApiV2Schema,
+                               schemaOptions: Set[SchemaOption],
                                requestingUser: UserADM): Future[ReadResourcesSequenceV2] = {
 
         // eliminate duplicate Iris
         val resourceIrisDistinct: Seq[IRI] = resourceIris.distinct
 
+        // Find out whether to query standoff along with text values. This boolean value will be passed to
+        // ConstructResponseUtilV2.makeTextValueContentV2.
+        val queryStandoff: Boolean = SchemaOptions.queryStandoffWithTextValues(targetSchema = targetSchema, schemaOptions = schemaOptions)
+
         for {
 
             queryResultsSeparated: Map[IRI, ResourceWithValueRdfData] <- getResourcesFromTriplestore(
                 resourceIris = resourceIris,
                 preview = false,
                 propertyIri = propertyIri,
                 versionDate = versionDate,
+                queryStandoff = queryStandoff,
                 requestingUser = requestingUser
             )
 
```

Comment:
Shouldn't getting the mappings depend on `queryStandoff`? If no standoff is to be queried, no mapping will be needed.

Reply:
or does `queryResultsSeparated` just does not contain any standoff in that case and nothing is done?